setup.py

```diff
@@ -0,0 +1,16 @@
+import json
+from pathlib import Path
+import os
+import shutil
+
+def setup():
+    b_profile = input("Please enter filename of bash profile: ").strip()
+    settings = {"BASH_PROFILE": b_profile}
+    with open("config.json", "w+") as config:
+        json.dump(settings, config)
+    shutil.move(os.getcwd(), str(Path.home()))
```

Comment:
This line causes an exception if the working directory is already in the home directory. I suggest just putting some sort of wrapper like this:

    HOME = str(Path.home())
    CUR_DIR = os.getcwd()
    if CUR_DIR != HOME + '/bash-manager':
        shutil.move(CUR_DIR, HOME)
        print("\nbash-manager has been moved to ~/bash-manager\n")

Reply:
Awesome, I'll get right on that. Thanks for accepting!